index.ts

```diff
@@ -158,10 +159,38 @@ function resetLastTouch() {
 let endPlayTime = -1;
 let endPlayTime2 = -1;
 
-// assume magnitude is between 0 and 1
+interface rangeArgs {
+  inputMin: number;
+  inputMax: number;
+  outputFloor: number;
+  outputCeil: number;
+};
+function generateRange(args: rangeArgs) {
```

Comment:
made this handy guy to map two ranges to eachother and get one-off results for an input val